CHANGELOG.rst

```diff
@@ -1,5 +1,29 @@
 Changelog
 =========
+1.4.0 (2019-10-30)
+-----------------------
+MLflow 1.4.0 includes several major features:
+
+- Model Registry. MLflow 1.4 adds an experimental model registry feature, where you can manage, version, and keep lineage of your production models. (#1943, @mparkhe, @Zangr, @sueann, @dbczumar, @smurching, @gioa, @clemens-db, @pogil, @mateiz)
```

Comment:
It might be better to call the registry beta so people know that it's still early on.